iac-extensions/docker/src/test/resources/checks/ClearTextProtocolDowngradeCheck/curl.dockerfile

```diff
@@ -32,6 +32,9 @@ RUN curl -foo -L -foo --proto "=foobar" https://redirecttoinsecure.example.com
 # Noncompliant@+1
 RUN curl --proto "=foobar" -foo -L -foo https://redirecttoinsecure.example.com
 
+# Noncompliant@+1
+RUN curl --tlsv1.2 -sSf -L https://might-redirect.example.com/install.sh
```

Comment:
Please add examples:
* `RUN curl --tlsv1.2 -sSf --location https://might-redirect.example.com/install.sh` (--location instead -L)
* more examples with `--location` flag
* `RUN curl --tlsv1.2 -sSfL https://might-redirect.example.com/install.sh` (from man page: "Short version options that do not need any additional values can be used immediately next to each other, like for example you can specify all the options -O, -L and -v at once as -OLv.")
* other examples with options containing `L` flag specified at once: `-LOv`, `-OLv`, `-vL` etc.

If it is hard to have support for such flags at once, please create a ticket.

Reply:
I managed to add support for grouped flags and even got some new results in the ITs. Maybe this is something that can also be moved to the CommandDetector in the future

Reply:
IMO, it makes sense to add support for grouped flags to the command detector as many commands support that. Please create a ticket for that improvement.

Reply:
https://sonarsource.atlassian.net/browse/SONARIAC-1483